extensions/3DTILES_bounding_volume_S2/README.md

```diff
@@ -0,0 +1,459 @@
+
+|Level 0|Level 1|
+|:-:|:-:|
+| ![Plane - Level 0](figures/plane_parent.png) S2 cell (`"1"`) on the Earth cube| ![Plane - Level 1](figures/plane_children.png) Children of S2 cell (`"1"`) on the Earth cube |
+| ![Ellipsoid - Level 0](figures/ellipsoid_parent.png)  S2 cell (`"1"`) on the WGS84 ellipsoid| ![Ellipsoid - Level 1](figures/ellipsoid_children.png) Children of S2 cell (`"1"`) on the WGS84 ellipsoid|
+
+The S2 library uses a modified Hilbert curve to provide a one dimensional ordering of cells on the S2 Earth cube. This provides each cell, from level 1 to level 30, with a unique 64-bit identifier. Using S2 cell IDs, centimeter scale areas be uniquely identified.
+
+| S2 Curve on Earth cube |  S2 Curve on WGS84 ellipsoid |
+|:-:|:-:|
+| ![Math](figures/plane.png)  | ![Math](figures/ellipsoid.png)  |
+
+## Cell IDs
+
+The 64-bit S2 cell ID is constructed as follows:
+
+1. Use 3 bits to encode the index of the root cell it belongs to. Valid values are in the range `[0-5]`.
+2. For a cell at level `k`, for each of the `k` "child" values, add 2 bits to the right, indicating the selection of one of 4 children during subdivision.
+3. Set the bit following the last child to `1`.
+4. Set the remaining bits to `0`.
+
+For example:
+
+```
+0011000000000...000   Root cell 2
+0010110000000...000   2nd child of root cell 2
+0010111100000...000   3rd child of 2nd child of root cell 2
+0010111001000...000   1st child of 3rd child of 2nd child of root cell 2
+```
+
+In their decimal forms, the cell IDs above are represented as follows:
+
+```
+3458764513820540928   Root cell 2
+3170534137668829184   2nd child of root cell 2
+3386706919782612992   3rd child of 2nd child of root cell 2
+3368692521273131008   1st child of 3rd child of 2nd child of root cell 2
+```
+
+## Tokens
+
+To provide a more concise representation of the cell ID, as well as to provide a better indication of the level of the cell, we can use the hexadecimal form of the cell ID and remove any trailing zeros to obtain the cell's token.
+
+For the cell IDs in the example above, the tokens are:
+
+```
+3     Root cell 2
+2c    2nd child of root cell 2
+2f    3rd child of 2nd child of root cell 2
+2ec   1st child of 3rd child of 2nd child of root cell 2
+```
+
+## Bounding Volume
+
+An S2 cell describes 4 positions on the surface of the WGS84 ellipsoid. The `minimumHeight` and `maximumHeight`, provided in meters, describe the minimum and maximum heights above (or below) each corner position of the cell respectively. A tile's [`transform`](../../specification#tile-transforms) property will be ignored when this extension is used for describing a tile's `boundingVolume`. Tiles using this extension must maintain [spatial coherence](../../specification/README.md#bounding-volume-spatial-coherence). This extension may be applied to the [`tile.boundingVolume`](../../specification/schema/tile.schema.json) or the [`content.boundingVolume`](../../specification/schema/content.schema.json) objects.
```

Comment:
@sanjeetsuhag is this rewording accurate? The top surface of an S2 bounding cell has a constant height not just at the corners but even in the interior, right?

```suggestion
An S2 cell describes 4 positions on the surface of the WGS84 ellipsoid, indicating the horizontal position of the corners of the bounding volume. The `minimumHeight` and `maximumHeight`, provided in meters, describe the heights of the bottom and top surface of the bounding volume, respectively. A tile's [`transform`](../../specification#tile-transforms) property will be ignored when this extension is used for describing a tile's `boundingVolume`. Tiles using this extension must maintain [spatial coherence](../../specification/README.md#bounding-volume-spatial-coherence). This extension may be applied to the [`tile.boundingVolume`](../../specification/schema/tile.schema.json) or the [`content.boundingVolume`](../../specification/schema/content.schema.json) objects.
```

Reply:
I am not sure if the term "horizontal" is helping or hurting here. I think the concept we want to convey here is that of "extrusion", and not "construction".

Reply:
I tried rewording this a different way, describing it as extrusion

Reply:
Latest description looks good to me.